src/neptune_scale/core/metadata_splitter.py

```diff
@@ -181,7 +180,7 @@ def _skip_non_finite(self, step: Optional[float | int], metrics: Dict[str, float
             v = float(v)
 
             if not math.isfinite(v):
-                if SKIP_NON_FINITE_METRICS:
+                if self._should_skip_non_finite_metrics:
                     logger.warning(
                         f"Skipping a non-finite value `{v}` of metric `{k}` at step `{step}`. "
                         f"You can turn this warning into an error by setting the `{envs.SKIP_NON_FINITE_METRICS}` "
```

Comment:
Can we show this instruction only once? It fills the logs without adding any context
![image](https://github.com/user-attachments/assets/7b3b3d27-922e-449f-a97f-ec15db05f538)